wire-runtime/src/test/java/com/squareup/wire/protos/oneof/OneOfMessage.java

```diff
@@ -21,8 +21,7 @@
    */
   @ProtoField(
       tag = 1,
-      type = Message.Datatype.INT32,
-      label = Message.Label.ONE_OF
```

Comment:
I'm going to wait for your `@ProtoField` runtime changes to land. Right now it's in an awkward intermediate state where we have `OPTIONAL` and `REQUIRED` but they don't really apply to proto3.